atomic_reactor/plugins/post_export_operator_manifests.py

```diff
@@ -118,8 +118,15 @@ def run(self):
                            IMG_MANIFESTS_PATH, ex)
             self.log.error(msg)
             raise RuntimeError('%s %s' % (msg, ex))
+
+        except Exception as ex:
+            raise RuntimeError('%s ' % (ex))
```

Comment:
Do we need to catch exception here?

Also:
* extra trailing whitespace in `'%s '`
* unneeded parenthesis in `(ex)`

Reply:
I'd say it doesn't hurt